provisioning/device/samples/ProvisioningDeviceClientX509/readme.md

```diff
@@ -42,8 +42,7 @@ to
 ```C# 
     var myCertificate = new X509Certificate2("myCertificate.pfx", "mypassword");
     var myChain = new X509Certificate2Collection();
-    myChain.Import("myChain.p7b");
-    using (var security = new SecurityProviderX509Certificate(myCertificate, myChain)) 
+    var security = new SecurityProviderX509Certificate(myCertificate, myChain);
```

Comment:
1. It's important to Dispose of this object (why the `using` statement was originally there).
2. `myChain` is empty after this change. This is fine as long as the certificates are correctly installed in your Intermediate Authorities store.

Reply:
1. Noob question: Why is it important to Dispose the object?
2. How do I check that the certs are correctly installed?

Reply:
> Why is it important to Dispose the object?

Disposing ensures that resources are de-allocated immediately. Otherwise GC will collect later or this will be performed at process exit.

> How do I check that the certs are correctly installed?

Verify the Intermediate Certificate Authorities store using certmgr.msc in Windows. In Linux, .NET Core provides a similar store construct that can only be accessed via .NET APIs.
![image](https://user-images.githubusercontent.com/8635911/36448306-0b7c8dcc-163c-11e8-988d-7b0260326f96.png)